swagger.yaml

```diff
@@ -431,7 +431,7 @@ components:
           example: sample_project
         created_at:
           type: integer
-          format: timestamp
+          format: string
           example: 1
         last_modified:
           type: integer
```

Comment:
Wait, so those are integers or string?

Reply:
i handle them as strings ( that means all the conversion is done on the server side, using serialize/deserialize methods visible in the second PR), do we want to have them as integers (int64) anyway?

Reply:
No, it is fine this way.